web/app/themes/ppj/functions.php

```diff
@@ -6,7 +6,14 @@ function enqueue_scripts()
     $legName  = getLegNameFromPath();
     $root_dir = get_template_directory_uri() . '/dest';
     wp_enqueue_style( 'main-css', $root_dir . mix_asset( "/css/{$legName}.css" ), null, null );
+
+    wp_enqueue_script( 'header-js', $root_dir . '/js/header.js', null, null, false );
```

Comment:
Do `header.js` and `main.js` need to be separate files, or could they be merged? It'd be one less HTTP request on each page load.

Reply:
Done